static/app/components/panels/panelItem.tsx

```diff
@@ -1,16 +1,27 @@
 import styled from '@emotion/styled';
-import {Flex} from 'reflexbox'; // eslint-disable-line no-restricted-imports
 
-const PanelItem = styled(Flex)`
+import space from 'app/styles/space';
+
+type Props = {
+  /**
+   * Disables the default padding
+   */
+  noPadding?: boolean;
+  /**
+   * Align items vertical center (assuming flex-direction isn't changed).
+   */
+  center?: boolean;
```

Comment:
just a suggestion :) 

```suggestion
  isCentered?: boolean;
```